main.py

```diff
@@ -45,118 +47,118 @@ def shutdown(self):
 
         If you have some cleanup procedure, insert it here.
         """
+        self.scheduler.shutdown()
 
     @rest('/v1', methods=['GET'])
+    def get_all_mw(self):
+        """Return all maintenance windows."""
+        maintenances = self.scheduler.list_maintenances()
+        return current_app.response_class(
```

Comment:
I used pydantics jsonfiy specifically to match `TIME_FMT` we are using. If we remove the requirement to match `TIME_FMT`, then we can perform marshalling with flask. This should also fix the issue with submitting times that don't include tz info.

Reply:
I see. So, this won't be a blocker for this PR, since the `TIME_FMT` with `%z` was already in place historically, but since we're stabilizing the v1 of this NApp, I'll encourage you to continue this discussion, supporting timezones other than UTC with the `%z` offset fmt can really complicate things and make it harder to maintain and also to correlate events (good thing that on logs we'll have UTC on each log entry). Maybe we can nuke `pytz` and only support UTC in the v1 of this NApp? It would be great to confirm and have a discussion with network operators. In the old blueprint draft (that will be refined) on issue https://github.com/kytos-ng/maintenance/issues/59 we also didn't have any requirement to support arbitrary timezones. 

We've been using UTC everywhere on NApps and core (core also natively supports encoding datetime with this [`"%Y-%m-%dT%H:%M:%S` format](https://github.com/kytos-ng/kytos/blob/master/kytos/core/api_server.py#L33).

Reply:
Fascinating discussion, guys. I've created issue #66 to continue the discussion around this topic!